frontend/package.json

```diff
@@ -2,7 +2,7 @@
     "name": "vantaa-pwa-dev",
     "version": "0.1.0",
     "private": true,
-    "proxy": "http://localhost:5000",
+    "proxy": "http://vantaa-pwa",
```

Comment:
With this the development only works within Docker.

Reply:
yes, indeed. What's wrong with that?

Reply:
Well, it causes usually more problems that it solves for actively developed services..

Reply:
:D well, i can tell you with confidence that this is more streamlined than poking around into different part of the applications and run various scripts.

Reply:
Yet again. That is not the point here. It should not be locked down that the Docker is only choice. It was already possible earlier to use that if you want.